website/content/en/docs/tutorials/iperf.md

```diff
@@ -164,23 +164,78 @@ deployment "cl-dataplane" successfully rolled out
 
 ## Enable cross-cluster access
 
-In this step, we enable connectivity access between the iPerf3 client and server.
+In this step, we enable connectivity access between the iPerf3 client and server.<br>
+For each step, you have an example demonstrating how to apply the command from a file or providing the complete CRD associated with the command.
+### Peers setup
 
-1. First, add the peers to each cluster:
+Add the peers to each cluster:
```

Comment:
nit: there is only one peer being added to each, so use singular `peer`, not `peers`
```suggestion
Add the remote peer to each cluster:
```